crates/storage/db/benches/iai.rs

```diff
@@ -76,3 +61,18 @@ impl_iai!(
     Config,
     SyncStage
 );
+
+/// Returns bench vectors in the format: `Vec<(Key, EncodedKey, Value, CompressedValue)>`.
+fn load_vectors<T>() -> Vec<(T::Key, bytes::Bytes, T::Value, bytes::Bytes)>
+where
```

Comment:
Is the next step here to make the vectors look realistic? I would perhaps try to generate them at runtime via some key/value distribution, e.g. say it's uniform, and assign a bunch of monotonic keys with uniform values, and then some random non-repeating keys with unfirom values too or something like that. feel free to get creative. and we can then add test-vectors to disk so that benchmarks don't change every time.